compiler/optimizer/VPConstraint.hpp

```diff
@@ -961,6 +961,7 @@ class VPKnownObject : public TR::VPFixedClass
 
    virtual TR_YesNoMaybe isJavaLangClassObject();
```

Comment:
The copyright date needs to be updated in this file

Reply:
fixed